sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -285,6 +292,38 @@ export interface KeyReleasePolicy {
     data?: Uint8Array;
 }
 
+// @public
+export interface KeyRotationLifetimeActions {
+    action?: "Rotate" | "Notify";
+    trigger?: KeyRotationLifetimeActionsTrigger;
+}
+
+// @public
+export interface KeyRotationLifetimeActionsTrigger {
```

Comment:
This also doesn't match our existing nameing for certificates' `LifetimeAction`, which we should try to mimic as best as possible. We could probably also drop the repeating `KeyRotation` prefix, though it would help disambiguate if both keys and certs were in scope. Whether we keep the prefix or not, I suggest `KeyRotationLifetimeActions` and `KeyRotationLifetimeAction`.

Reply:
We may also want to just flatten these, which is what we did for certificates' `LifetimeAction`.

Reply:
Can you tell I've done hardly any work on certificates since joining the team? 😆 I honestly (and shamefully) didn't even realize CertificatePolicy was a thing.

Thanks for the feedback - I am reworking things to match CertificatePolicy more closely. Not quite there with my latest commit, but closer I think.